BE/src/main/java/com/example/User/EnergyUpdateRequest.java

```diff
@@ -0,0 +1,13 @@
+package com.example.User;
+
+public class EnergyUpdateRequest {
+    private int energy;
+
+    public int getEnergy() {
+        return energy;
+    }
+
+    public void setEnergy(int energy) {
+        this.energy = energy;
+    }
+}
```

Comment:
DTOs should be immutable collections. So, change this class's type like so and remove the getters / setters:
```suggestion
public record EnergyUpdateRequest(Integer energy) {}
```